compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -347,52 +351,61 @@ void addPadPackBasedPassPipeline(OpPassManager &pm,
     AMDAIEBufferizeToAllocationOptions bufferizeOptions2;
     bufferizeOptions2.memorySpace = 2;
     bufferizeOptions2.bufferizeOperand = BufferizeOperand::Input;
-    modulePassManager.addNestedPass<func::FuncOp>(
+    modulePassManager.addPass(
         createAMDAIEBufferizeToAllocationPass(bufferizeOptions2));
   }
-
+  /*
   // Lower to UKernels
   {
     AMDAIELowerToUKernelsOptions options;
     options.passPipeline = AIEPassPipeline::PadPackPipeline;
     options.pathToUkernels = clPathToUkernels;
-    modulePassManager.addNestedPass<func::FuncOp>(
+    modulePassManager.addPass(
         createAMDAIELowerToUKernelsPass(options));
   }
-
+  */
   // Vectorization passes
   appendVectorizationToPipeline(modulePassManager);
 
   // Comprehensive bufferization
   addAMDAIEBufferizePasses(modulePassManager);
-  modulePassManager.addPass(createLowerUKernelOpsToCallsPass());
+  //modulePassManager.addPass(createLowerUKernelOpsToCallsPass());
```

Comment:
Move this out of here just before invoking the conversion-to-air-to-aie pass pipelines.